Calculator.Logic/Operation.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Calculator.Logic
+{
+    public enum Operation
+    {
+        None = 0,
+        Sum = 1,
+        Subtract = 2,
+        Multiply = 3,
+        Divide = 4,
+        Power = 5,
+        IsPrime = 6,
+        IsOddOrEven = 7,
+        SquareRoot = 8,
+        AbsoluteValue = 9,
+        Reverse = 10,
+        Palindrome = 11,
```

Comment:
Use some sort of naming convetion, if you called IsPrime and IsOddOrEven then call these IsPalindrome/IsSuperpalindrome